packages/orchestrator/internal/template/build/commands/copy_script.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
```

Comment:
Using `env` is probably more portable, if you want to make another change here

```suggestion
#!/usr/bin/env bash
```